SAKPaaS/src/app/components/user-card/user-card.component.scss

```diff
@@ -47,6 +47,7 @@
 
   &.expanded {
     width: 440px;
+    min-width: auto;
```

Comment:
Dadurch wird die Animation wieder unschön

Reply:
Nope, because `width` is still there. But it also doesn't bring the benefits I expected, so will remove it.